docs/setup/compute-ntp.md

```diff
@@ -35,6 +34,12 @@ If your Create® 3 and compute board have an internet connection, this is not re
 
         sudo service chrony restart
 
+1. Log into the Create® 3 web application and modify the NTP sources (as of Jul 2023 under "Beta Features") to add the following: 
```

Comment:
I think these steps were omitted from the guide, but I needed to do this in order for the create 3 to start syncing time with the raspi